main.go

```diff
@@ -53,6 +53,11 @@ func main() {
 
 	kiln.LogInfo.Printf("ImageCreator set up.")
 
+	clusterConfig, err := kiln.NewClusterConfig()
```

Comment:
I made a function to get a local config. I'll follow the config for this that y'all are doing.